docs/devguide/modules-dev-guide.asciidoc

```diff
@@ -336,6 +354,88 @@ Note that you should follow the convention of naming of fields prefixed with the
 module and fileset name: `{module}.{fileset}.field`, e.g.
 `nginx.access.remote_ip`. Also, please review our <<event-conventions>>.
 
+In 6.6 and later, ingest pipelines can use the `pipeline` processor to delegate
+part of the processing to another sub-pipeline.
+
+[source,json]
+----
+{
+  "pipeline": {
+    "name": "name-of-sub-pipeline"
+  }
+}
+----
+
+This can be useful if you want a fileset to ingest the same _logical_ information
+presented in different formats, e.g. csv vs. json versions of the same log files.
+Imagine a root ingest pipeline that detects the format of a log entry and then conditionally
```

Comment:
We are already introducing the "entry point pipeline" terminology further up in this doc. So could we say the following here?

> Imagine an entry point ingest pipeline that detects the format of a log entry and then conditionally delegates further processing of that log entry, depending on the format, to other
pipelines.

Reply:
Since you describe the behavior of the pipeline, you can just say "ingest pipeline" here.

IMO, fewer words = easier to parse = quicker to read = happier user.

I don't object strongly, though, so please proceed as you see fit.